.github/workflows/ci.yml

```diff
@@ -49,27 +43,33 @@ jobs:
       artifacts_to_cache: ${{ steps.get_artifacts_to_cache.outputs.artifacts_to_cache }}
     # runs-on: ubuntu-latest
     runs-on: ['self-hosted', 'org', '8-cpu']
-    container:
-      image: us-west1-docker.pkg.dev/devopsre/actions-runner-controller/celo-monorepo:node18
```

Comment:
Would be good to check whether not using this container helps. If it does, we might want to rework it.

Reply:
the container has a lot of stuff we dont need and its init time was very long. eventually a container with just node (with nodes corepack feature enabled) would be perfect.